_content/monographs/so-it-was-said_sujato.yaml

```diff
@@ -0,0 +1,22 @@
+---
+title: "So It Was Said: A Delectable Translation of the Itivuttaka"
+authors:
+  - sujato
+external_url: "https://suttacentral.net/edition/iti/en/sujato?lang=en/"
+drive_links:
+  - "https://drive.google.com/file/d/1j3cbmmDMQCVXLmnEhWwjFPxXSP_3DaLV/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1nI0nZaYl_qrZ84Z5CQUNLQY-7CH_0Z_A/view?usp=drive_link"
+formats: [pdf, epub]
+course: iti
+tags:
+  - sutta
+year: 2020
+publisher: "Sutta Central"
```

Comment:
Nix the publisher and move this to booklets. Also: the correct file extension for these is `md` (not `yaml`)